src/styles/blocks/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+  margin-right: $card-margin-right;
+  margin-bottom: 46px;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__image {
+    box-sizing: content-box;
+    height: 134px;
+    width: 160px;
+    padding: 0 3px;
+
+  }
+
+  &__name-of-product {
+    width: 166px;
+    font-weight: 500;
+    padding-top: 38px;
```

Comment:
Paddings looks strange as for me. Padding increases elements size, it can be use for example to increase clickable zone, or size of background, but to set distance is more clear to use margin